src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -3793,7 +4166,7 @@ public PropertyDescriptorCollection GetProperties(Attribute[]? attributes)
             /// <summary>
             /// ICustomTypeDescriptor implementation.
             /// </summary>
```

Comment:
NIT:
```suggestion
```

Reply:
I was following the comment pattern elsewhere in this file.